src/Symfony/Component/Lock/Store/StoreFactory.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Lock\Store;
+
+use Symfony\Component\Lock\Exception\InvalidArgumentException;
+
+/**
+ * StoreFactory create stores and connections.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+class StoreFactory
+{
+    public static function createConnection($dsn, array $options = array())
```

Comment:
I think we should add a docblock listing supported dsn

Reply:
will be removed replaced by the new Dns component